Homework 3 completed

nechepurenkoN opened the pull request.

nechepurenkoN committed:
hw3 is done

nechepurenkoN committed:
move versions to property section and add a new line in properties file

shoggy commented on pom.xml:
```diff
@@ -82,6 +82,24 @@
+      <artifactId>lombok</artifactId>
+      <version>${lombok.version}</version>
+      <scope>compile</scope>
+    </dependency>
```
Duplicated dependencies

shoggy commented on src/main/java/utils/JdiTestingIndexPageData.java:
```diff
@@ -0,0 +1,77 @@
+
+    public static final String PASSWORD = getPassword();
+
+    public static final String USERNAME = "Roman Iovlev";
```
good candidate for externalization, like login/password

shoggy reviewed (changes requested):
Please fix Compilation Errors

nechepurenkoN committed:
add user pojo and fix deps in pom file

nechepurenkoN commented:
@shoggy I changed scope of those libs (`testng`, `lombok`, `webdrivermanager`) to default (`compile`) in order to use them in `src` files. I fixed compilation problems after I replaced a file from `hw2`,  now `hw2` package is in a correct state and project compiles without errors.

Also I add User pojo to encapsulate user credentials.
